apps/nextra/pages/en/build/smart-contracts/book/enums.mdx

```diff
@@ -0,0 +1,214 @@
+# Enums
+
+_Since language version 2.0_
+
+Enum types are similar to struct types but support defining multiple *variants* of the data layout. Each variant has its distinct set of fields. In expressions, tools for testing, matching, and deconstructing enum variants are supported. 
+
+
+## Declaration of Enum Types 
+
+An enum type declaration lists the number of different variants, as seen in the example below:
+
+```move
+enum Shape {
+    Circle{radius: u64},
+    Rectangle{width: u64, height: u64}
+}
+```
+
+There can be zero or more fields for an enum variant. If no arguments are given, the braces can also be omitted, declaring simple values:
+
+```move
+enum Color {
+  Red, Blue, Green
+}
+```
+
+Similar to struct types, enum types can have abilities. For example, for the `Color` enum type it is more appropriate to make it copyable, droppable, and storable, similar like primitive number types:
+
+```move
+enum Color has copy, drop, store, key { ... }
```

Comment:
```suggestion
enum Color has copy, drop, store, key { Red, Blue, Green }
```

Reply:
Because `..` is a new syntax we are also introducing, having `...` may lead readers to think that is also some new syntax.